### PR TITLE
FACES-3173 - Exclude **/internal/** packages from javadoc generation

### PR DESCRIPTION
**Pull request by vsingleton**
This should be forward ported unless it breaks the build because of FACES-3192

### CODE REVIEW

File: bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 /**
  * This class provides the ability to parse the {@link StringBundler} value of the LIFERAY_SHARED_PAGE_TOP request
- * attribute and remove duplicate CSS and/or JavaScript resources that are meant to be rendered in the <head>...</head>
+ * attribute and remove duplicate CSS and/or JavaScript resources that are meant to be rendered in the 'head'
```
Comment (ngriffin7a) [inline bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java:36]: @vsingleton Please try `&lt;head&gt;` or `&amp;lt;head&amp;gt;` instead and let me know if either one works.

Comment (ngriffin7a) [reply bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java:36]: @vsingleton NVM, I tried `&lt;head&gt;` and it worked. I'll make a code review commit.

**Comment by ngriffin7a** [comment]
See code review commit https://github.com/liferay/liferay-faces-bridge-ext/commit/e6d5dc8c4732746cec1640bd19f183564002f845 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ngriffin7a** [comment]
Merged, thank you. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java (hunk 1 above)
Comment (vsingleton) [reply bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java:36]: @ngriffin7a One place I used the &lt; for a fix but not for this <head> issue.  I was inconsistent.  Thanks for catching this.

**Pull request closed without merging**